src/development/troubleshoot.md

```diff
@@ -2,6 +2,14 @@
 
 <!-- toc -->
 
+## Force a redeploy
+
+There are times where you might want to trigger a redeployment of your application. As with every other action on Platform.sh, this requires a git commit, however this can be empty. It is not recommended to do this if there are builds in a pending state, as these will block deployment. Wait for those builds to complete.
+
+```
+$ git commit --allow-empty -m 'force redeploy'
+$ git push platform <branch-name>
+```
```

Comment:
Should we also mention the set-a-variable method?  (I kinda like that alternative as it keeps a clean git history.)

Reply:
Fair point. Will add.

Reply:
my version of this is `platform vset redeploy "$(date)"`  so people don't start messing with test-1 test-2 etc..